resources/dicts/events/misc_events/general/general.json

```diff
@@ -141,23 +151,28 @@
      },
      {
         "camp": "any",
+        "tags": ["Leaf-bare", "other_cat", "other_cat_mate", "to_both", "platonic", "romantic"],
         "tags": ["Leaf-bare", "other_cat",  "other_cat_mate", "to_both", "platonic", "romantic"],
```

Comment:
am i just seeing things or have some lines been duplicated?

Reply:
What in the world. Okay fixing it lol.